include/temp.h

```diff
@@ -0,0 +1,15 @@
+#ifndef __TEMP_H__
+#define __TEMP_H__
+
+#include "mbed.h"
+
+// Steinhart-Hart equation constants. Equation: 1/T = (1/T0)+(1/B)ln(R/R0)
+struct Therm_Constants{
+  int B_CONSTANT;
+  int R_CONSTANT;
+  int T_CONSTANT;
+};
+
```

Comment:
Please add a comment about what **exactly** this function is for. Really important since it is now designed super specifically for the thermistor on the board